templates/security/certifications.html

```diff
@@ -38,7 +38,7 @@ <h2 class="p-heading--3">Run regulated and high security workloads on Ubuntu</h2
  <div class="row p-divider">
    <div class="col-4 p-divider__block">
      <h3>Comply with established security baselines</h3>
-     <p>The default configuration of Ubuntu balances usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. Canonical works with DISA to ensure STIG guides are available for Ubuntu, as well as provides OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/pro">Ubuntu Advantage</a> and <a href="/cloud/public-cloud">Ubuntu Pro</a>.</p>
+     <p>The default configuration of Ubuntu balances usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. Canonical works with DISA to ensure STIG guides are available for Ubuntu, as well as provides OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/pro">Ubuntu Pro on-premise</a> or on <a href="/public-cloud">Public Clouds</a>.</p>
```

Comment:
```suggestion
     <p>The default configuration of Ubuntu balances usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. Canonical works with DISA to ensure STIG guides are available for Ubuntu, as well as provides OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/pro">Ubuntu Pro on-premise</a> or on <a href="/public-cloud">public clouds</a>.</p>
```
@juanruitina 's left a comment requesting that this not be capitalized in different section. I'd imagine that that convention extends here as well, but I left a comment on the doc just in case